xlml/utils/xpk.py

```diff
@@ -62,6 +62,9 @@ def run_workload(
       f"gcloud config set compute/zone {zone}",
       "git clone https://github.com/google/xpk.git /tmp/xpk",
       "cd /tmp/xpk",
+      "apt-get update && apt-get install -y google-cloud-sdk",
+      "apt-get install -y kubectl",
```

Comment:
Have you tested it? If so, we could just remove those dependencies from example Dockerfile.

Reply:
@raymondzouu we should move as much environment prep and dependency installation into the docker image as possible, but since XPK support is still WIP this approach is fine for now.

Reply:
I tested this and it worked, thanks!